polkadot/xcm/pallet-xcm-benchmarks/src/fungible/benchmarking.rs

```diff
@@ -354,7 +328,6 @@ benchmarks_instance_pallet! {
 	}: {
 		executor.bench_process(xcm)?;
 	} verify {
-		assert!(T::TransactAsset::balance(&sender_account) <= sender_account_balance_before);
```

Comment:
Can we at least check/assert that `executor.holding()` is changed? Should it change all the time?